src/Common/src/System/IO/DelegatingStream.cs

```diff
@@ -172,6 +172,11 @@ public override void EndWrite(IAsyncResult asyncResult)
             _innerStream.EndWrite(asyncResult);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
+        {
+            _innerStream.CopyTo(destination, bufferSize);
+        }
```

Comment:
Looks like this file is also built for netfx.  Do we ever actually expose this stream anywhere where CopyTo might be called on it?  If not, probably simplest to just revert this change, too.